File: backend/internal/constants/phase_disposition.go
package constants

type PhaseDisposition string

const (
	DispositionAcceptEvidence  PhaseDisposition = "accept_evidence"
	DispositionRequestFollowUp PhaseDisposition = "request_follow_up"
	DispositionFalsePositive   PhaseDisposition = "dismiss_false_positive"
)

func (d PhaseDisposition) Valid() bool {
	switch d {
	case DispositionAcceptEvidence, DispositionRequestFollowUp, DispositionFalsePositive:
		return true
	default:
		return false
	}
}

func PhaseDispositionValues() []string {
	return []string{
		string(DispositionAcceptEvidence), string(DispositionRequestFollowUp), string(DispositionFalsePositive),
	}
}

File: frontend/src/types/enums/phase-disposition.ts
export const phaseDispositions = ['accept_evidence', 'request_follow_up', 'dismiss_false_positive'] as const
export type PhaseDisposition = typeof phaseDispositions[number]
export const phaseDispositionLabels: Record<PhaseDisposition, string> = {
  accept_evidence: '认可证据',
  request_follow_up: '要求补查',
  dismiss_false_positive: '排除误报',
}
